README.md

```diff
@@ -57,6 +53,33 @@ To run the a11y audit locally
 
 *To view visit http://localhost:8000/demos/<name>*
 
+## Testing for Accessibility
+
+PatternFly uses [aXe: The Accessibility Engine](https://www.deque.com/axe/) to check for accessibility violations. Our goal is to meet WCAG 2.0 AA requirements, as noted in our [Accessibility Guide](https://pf-next.com/accessibility-guide).
+
+### How to Perform an Accessibility Audit with aXe
+aXe is available as either a browser extension or npm script.
+
+To run the a11y audit locally:
+- install the latest [chromedriver](http://chromedriver.chromium.org/downloads) and ensure its available on your system `$PATH`
+  - alternatively, macOS users can simply `brew cask install chromedriver`
+- run `npm run dev`
+- run `npm run a11y` (in a separate terminal)
+
+The tool is configured to return WCAG 2.0 AA violations for the full page renderings of all components, layouts, utilities, and demos. The tool will provide feedback about what the violation is and a link to documentation about how to address the violation.
+
+The same tool is also available as a browser extension for [Chrome](https://chrome.google.com/webstore/detail/axe/lhdoppojpmngadmnindnejefpokejbdd) and [Firefox](https://addons.mozilla.org/en-US/firefox/addon/axe-devtools/). 
```

Comment:
Should we also add a link here for axe-coconut? Helps if users want to test against leading edge of axe-core which has experimental support for latest and greatest a11y rules, including support for things like shadow DOM

https://chrome.google.com/webstore/detail/axe-coconut/iobddmbdndbbbfjopjdgadphaoihpojp

Reply:
I'm on the fence on this. In the README, I'm inclined to only reference tools that we expect to be used as part of the contribution process. Can we chat outside this PR about the benefits of referencing this in the a11y guide?